src/handler/methods.rs

```diff
@@ -17,7 +16,7 @@ where
     where
         Asyncify<Pred>: Injectable<Input, bool, FnArgs> + Send + Sync + 'a,
     {
-        self.chain(filter(pred))
+        self.chain(crate::filter(pred))
```

Comment:
Is there a reason to use `crate::` prefixes here?

Reply:
Yes, otherwise you just duplicate handler names in two places: import paths and their respective invocation. I think that using `crate::x` is better here because these are used only once.

Reply:
Makes sense